R/misc.R

```diff
@@ -63,7 +63,7 @@ ilinkfun_raw <- function(x, link_nm) {
   return(basic_ilink(x))
 }
 
-auc <- function(x) {
+.auc <- function(x) {
```

Comment:
See my comment concerning `.select()` for why I personally prefer not to start function names with a dot.

Reply:
Again, I want a different name to not cause confusion with very popular pROC::auc()